src/gluonts/transform/split.py

```diff
@@ -269,16 +267,18 @@ def __init__(
         instance_sampler: InstanceSampler,
         instance_length: int,
         output_NTC: bool = True,
-        time_series_fields: List[str] = [],
+        time_series_fields: Optional[List[str]] = None,
         allow_target_padding: bool = False,
         pad_value: float = 0.0,
         use_prediction_features: bool = False,
         prediction_length: Optional[int] = None,
     ) -> None:
+        super().__init__()
+
         self.instance_sampler = instance_sampler
         self.instance_length = instance_length
         self.output_NTC = output_NTC
-        self.dynamic_feature_fields = time_series_fields
+        self.dynamic_feature_fields = time_series_fields or []
```

Comment:
If this is using `validated`, this might not be required, since `pydantic` always returns a `copy`.

Reply:
Mhh ok, pydantic is super weird :D